README.md

```diff
@@ -174,4 +174,6 @@ I would like to thank Microverse.
 
 This project is [MIT](./LICENSE) licensed.
 
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ Optional ] Please consider removing this instructional text from your readme file, this text serves to guide you on how to fill this section of the readme file. You are supposed to remove the instructional text after filling in this section, please remove it.